_specification-v1/components.md

```diff
@@ -26,7 +21,7 @@ Origami components are repositories containing front end code which can be used
 
 ## Origami.json manifest
 
-All Origami components **must** contain an `origami.json` file at the top of the repository's directory structure. The [`origami.json` manifest specification](/spec/v1/manifest/) covers the contents of this file. In addition to the rules outlined in the manifest specification, Origami components **must** set the `origamiType` property in the JSON to `"module"`.
```

Comment:
shouldn't this file be in _specification-v2? we don't want the v1 spec to link to the v2  spec do we??

Reply:
You are right, I moved them to the spec-v1 folder to make it possible to see a diff between spec v1 and spec v2.

I'm happy to have it moved back to the spec-v2 folder 👍

Reply:
ah cool. OK
i'll work on it there then